plugins/docs-tools/skills/docs-workflow-jira-ready/scripts/jira-ready-check.sh

```
@@ -60,17 +60,20 @@ fi
 
 # --- Validate environment ---
 if [[ -z "${JIRA_API_TOKEN:-}" ]]; then
-  # Try sourcing ~/.env
+  # Load global defaults, then local overrides
   set -a; source ~/.env 2>/dev/null || true; set +a
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)